lib/magma/project.rb

```diff
@@ -47,6 +49,31 @@ def load_project
         require_files('loaders')
         require_files('metrics')
       end
+
+      load_model_attributes
+    end
+
+    def load_model_attributes
+      model_attributes = Magma.instance.db[:attributes].
+        where(project_name: @project_name.to_s).
+        to_a.
+        group_by { |attribute| attribute[:model_name].to_sym }
+
+      model_attributes.each do |model_name, attributes|
+        model = models[model_name]
+        raise load_error(model_name, attributes) unless model
```

Comment:
I would prefer to move the logic in `#load_error` into `LoadError` so that we can `raise LoadError` here, not being able to see what error class you are raising is too opaque.

Reply:
Ok, I moved it in https://github.com/mountetna/magma/pull/132/commits/b94e8b1f5f927f7f9e8acb2b26d0275c47821a04. I also renamed it to `AttributeLoadError` because it better represents what the error is.